codecov.yml

```diff
@@ -0,0 +1,15 @@
+# Always comment on PRs
+comment:
+  layout: "diff, flags, files" # delete old and post new
+  behavior: new
+  require_changes: false
```

Comment:
why would we want to post a coverage when there are no changes?

Reply:
This is the default behavior, I just thought it's better to have it explicit. 

I don't have a strong opinion here, I can change this to `true`.

Reply:
Let's leave it if is the default behavior, I agree that is better to have the configurations explicit! 👍